roles/newrelic/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Download installer
+  ansible.builtin.get_url:
+    url: "https://download.newrelic.com/install/newrelic-cli/scripts/install.sh"
+    dest: "/tmp/new-relic-install.sh"
+    mode: "777"
+
+- name: Install New Relic command
+  ansible.builtin.command: "/tmp/new-relic-install.sh"
+  args:
+    creates: /usr/local/bin/newrelic
```

Comment:
Nice to see some validation of the result. 
I can think of a couple of other ways to validate: 
1. check for a non-error return code, or 
2. grep the output for the success message. 

But I don't think it's worth spending more effort on figuring one of those out.